kalite/knowledgemap/templates/knowledgemap/knowledgemap.html

```diff
@@ -0,0 +1,637 @@
+{% extends base_template %}
```

Comment:
This could do with some refactoring and clean up.

First, extracting the javascript into separate javascript files, then we can sit down and discuss how to wrap this in backbone to make it more portable.